httpService/pkg/user/service_test.go

```diff
@@ -0,0 +1,184 @@
+package user_test
+
+import (
+	"context"
+	"os"
+	"testing"
+
+	"github.com/go-kit/kit/log"
+	"github.com/stretchr/testify/assert"
+	"github.com/timoteoBone/project-microservice/grpcService/pkg/entities"
+	errors "github.com/timoteoBone/project-microservice/grpcService/pkg/errors"
+	"github.com/timoteoBone/project-microservice/httpService/pkg/user"
+	util "github.com/timoteoBone/project-microservice/httpService/pkg/utils"
```

Comment:
although the mock should be in another package and not inside util, you should name util with an alias corresponding to its function, something like mock